### DISCUSSION TITLE
cannot receive text from aideck on crazeflie though wifi mode

### DISCUSSION THREAD

**Discussion by Salmck** [bitcraze/discussions#762] [Q&A]
This code doesn't seem to work:
> cpxPrintToConsole(LOG_TO_CRTP, "Hello World\n");

I cannot receive "Hello World" from aideck.

cfclient console showed as follows:
> 3B002A and I have 1024KB of flash!
> CFGBLK: v1, verification 
> DECK_CORE: 1 deck(s) found
> DECK_CORE: Calling INIT on driver bcAI for deck 0
> CPX-EXT-ROUTER: Waiting for CPX External router initialization
> CPX-EXT-ROUTER: CPX External router initialized, CPX_VERSION: 0
> AIDECK: Not setting up WiFi
> IMU: BMI088: Using I2C interface.
> IMU: BMI088 Gyro connection .
> IMU: BMI088 Accel connection 
> IMU: BMP388 I2C connection 
> ESTIMATOR: Using Complementary (1) estimator
> CONTROLLER: Using PID (1) controller
> MTR-DRV: Using brushed motor driver
> SYS: About to run tests in system.c.
> EEPROM: I2C connection .
> STORAGE: Storage check .
> IMU: BMI088 gyro self-test 
> DECK_CORE: Deck 0 test .
> SYS: Self test passed!
> STAB: Wait for sensor calibration...
> SYS: Free heap: 14600 bytes
> GAP8......
> STAB: Starting stabilizer loop
> SUP: Ready to fly

crazeflie-firmware stocked at here:
> static void cpx(void* _param) {
>   systemWaitStart();
>   while (1) {
>     DEBUG_PRINT("GAP8......\n");
>     cpxInternalRouterReceiveOthers(&cpxRx);  <<-- HERE

Anyone can tell me how to solve it

**Comment by knmcguire** [comment]
Hi! Some questions first

Have you updated all firmware according to [the aideck tutorial](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/)? Does this happend with the wifi example or have you tried any others?
